website/src/components/Showcase/Button.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import styles from './Button.module.css';
+
+const Button = (props) => {
```

Comment:
Looks useless, you could just define a couple of CSS rules and render a simple `<button>` with them. Why define a class for such a trivial case?

Reply:
Hi Yarik, my apologies for this one. I meant to delete this component as it is useless and was used for testing purposes. Deleted it